cmd/kubeadm/app/apis/kubeadm/types.go

```diff
@@ -84,7 +84,12 @@ type Etcd struct {
 type NodeConfiguration struct {
 	metav1.TypeMeta
 
-	Discovery Discovery
+	DiscoveryURL      string
```

Comment:
Don't want to hold up the PR for this, but we should start documenting all of the options here like every other type.go in the project.